overlord/state/task.go

```diff
@@ -250,15 +249,15 @@ func (t *Task) Wait(resultStatus Status) {
 		// aborted so it can transition to Undo.
 		return
 	}
-	t.waitStatus = resultStatus
+	t.waitedStatus = resultStatus
 	t.changeStatus(old, WaitStatus)
 }
 
-// WaitStatus returns the status the Task should return to once the current WaitStatus
+// WaitedStatus returns the status the Task should return to once the current WaitedStatus
```

Comment:
the second WaitedStatus needs to be WaitStatus I think

Reply:
Ah right, the renaming got a bit to inclusive. Thanks for spotting this!